packages/broker-service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:22-slim
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN yarn install --frozen-lockfile
+
+RUN yarn workspace @zkdb/storage build
+
+RUN yarn workspace @zkdb/broker-service build
```

Comment:
`yarn install && yarn build && yarn install --production --frozen-lockfile`
After build, can we remove all `dev-dependencies` to make these images smaller?

Reply:
I will try later on this. Earlier I stuck on writing multi-stage build Dockerfile so I don't have time to try on this.